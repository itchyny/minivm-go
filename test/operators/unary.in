print -10
print -3.14
print +++5
print -----6.12
print !true
print !!true
