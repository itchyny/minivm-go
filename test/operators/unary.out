-10
-3.14
5
-6.12
false
true
